src/chrome/content/rules/Refunite.org.xml

```diff
@@ -0,0 +1,11 @@
+<!--
+	Nonfunctional hosts in *.refunite.org:
+
+	- www: mismatch
+-->
+<ruleset name="Refunite.org">
+	<target host="refunite.org" />
```

Comment:
you can add `www.refunite.org` as a target with the rule `<rule from="http://www\.refunite\.org/" to="https://refunite.org/" />`

Reply:
My bad, sorry, I made a typo:

<rule from="^http://www\.refunite\.org/" to="https://refunite.org/" />

Reply:
ah, almost added that myself, but thought I must be misreading it somehow ;)
Thanks!